anyway/parsers/location_extraction.py

```diff
@@ -86,19 +83,15 @@ def get_db_matching_location(latitude, longitude, resolution, road_no=None):
             lambda x: geod.Inverse(latitude, longitude, x["latitude"], x["longitude"])["s12"],
             axis=1,
         )
-        markers = markers.replace({pd.np.nan: None})
```

Comment:
@elazarg Have you verified this? When I put pd.np.nan I remember that it didn't work with np.nan.

Reply:
This fixes the deprecation warning issued by pandas, which asks to use numpy directly.
We might want to use [`fillna`](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.fillna.html) though. What do you think?

Reply:
Okay. Turns out neither `np.nan` nor `pd.np.nan` work here - probably because `nan != nan`. So we must use `fillna` (which does work).